### PR TITLE
Update test Django app & docker-compose services for protos

### PR DESCRIPTION
**Pull request by tayloraburgess**

### CODE REVIEW

File: tests/management/commands/proto_create_invoice.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        invoice_event = invoice_updated_pb2.InvoiceUpdatedEvent()
+        invoice_event.invoice.id = str(uuid.uuid4())
+
+        pubsub.publish_proto(invoice_event, env.get("SNS_ARN"))
+
+        print(f"Invoice Event published: {invoice_event.__str__()}")
```
Comment (mattbriancon) [inline tests/management/commands/proto_create_invoice.py]: nit: `str(invoice_event)` is the Pythonic way. Same below.

File: tests/settings.py [hunk 2]
```diff
@@ -5,6 +5,7 @@
 import rollbar
 
 from mbq import env, metrics
+from mbq.protos.invoicing.events import invoice_updated_pb2
```
Comment (mattbriancon) [inline tests/settings.py]: Maybe not for this PR but we should look at making this import cleaner. I'm imaging being able to do

```python
from mbq import protos

{
    protos.invoicing.events.InvoiceUpdatedEvent: "...",
}
```

Comment (tayloraburgess) [reply tests/settings.py]: Yeah, that looks better to me--are you suggesting that we remove the need to have the `*_pb2.py` files in the import path?

Comment (mattbriancon) [reply tests/settings.py]: Yeah I think we can add some code to the `__init__.py`s in the generated code.

**Pull request merged**